packages/react-core/src/components/Truncate/Truncate.tsx

```diff
@@ -14,49 +14,49 @@ const truncateStyles = {
   end: styles.truncateStart
 };
 
+const minWidthCharacters: number = 12;
+
 interface TruncateProps extends React.HTMLProps<HTMLSpanElement> {
   /** Class to add to outer span */
   className?: string;
   /** Text to truncate */
   content: string;
+  /** Number of characters that shoul be sliced at the end of the content string. Works best if the number is between 5 to 10. */
```

Comment:
Some suggested spelling adjustments:
"that *should be sliced"
"between 5 *and 10"

Also, I'm curious why a number between 5 and 10 would work the best. I'm thinking others might wonder the same thing reading this.

Reply:
Thanks for the spellcheck... And why numbers between 5 and 10, I am taking it from this PR https://github.com/patternfly/patternfly/pull/4599 I could probably just say that we suggest these numbers... It is not mentioned in the core, just in that PR, so I wanted to mention it somewhere :) I welcome any better formulation of this piece of information.

Reply:
I think specifying it's a suggestion for the sake of readability makes sense. Maybe the example text which echos the same thing could be updated with that verbiage as well.

Reply:
1 other suggestion is to rename the prop to something like `charGap` (or something like gap) to add some more context to what the slicing will do, which is create a gap of whitespace. Not a blocking comment, just figured I'd throw this out there to see what you or others thought.